test/integration/front_proxy_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/openshift/origin/pkg/api/legacy"
 	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
 	authorizationclient "github.com/openshift/origin/pkg/authorization/generated/internalclientset"
-	"github.com/openshift/origin/pkg/cmd/server/admin"
+	"github.com/openshift/origin/pkg/oc/cli/admin/server"
```

Comment:
how does this compile?

Reply:
oh, because I didn't try `make test` just `make`. updated